dataeng/jobs/analytics/WarehouseTransformsMasterCI.groovy

```diff
@@ -126,6 +129,7 @@ class WarehouseTransformsMasterCI{
                 upstream('warehouse-transforms-master-ci', 'SUCCESS')
             }
             publishers common_publishers(allVars)
+            publishers slack_publisher(allVars)
```

Comment:
Curious if this still works as the method signature doesn't match.

Reply:
Ohh, it worked actually but I probably need to remove this parameter.

Reply:
@HassanJaveed84 
here is the fixing PR: [https://github.com/edx/jenkins-job-dsl/pull/1359/files](https://github.com/edx/jenkins-job-dsl/pull/1359/files)